### DISCUSSION TITLE
Loading state forever Fastapi-Graphql

### DISCUSSION THREAD

**Discussion by danielahdez** [fastapi/fastapi#7777] [Questions]
Hi everyone. I’m  trying FastApi with GraphQL (this example  https://fastapi.tiangolo.com/advanced/graphql/) and it is not working for me. When I run the project with uvicorn server stays stuck in a loading state forever... I be looking similars issues but don’t see any solution for me. Someone told me it's because my pc is offline but the others examples (not graphql) run perfectly without internet... or I really need to be online? ... can anyone help me? In advance thanks and sorry for my english (speak spanish)

**Comment by phy25** [comment]
Could you post relevant environment information in a comment e.g. https://github.com/tiangolo/fastapi/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D ?

**Comment by danielahdez** [comment]
@phy25 I just did it 🙌

https://github.com/tiangolo/fastapi/issues/973#issue-563596709

Closing this one.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 :cake: :bowing_man: 

And thanks @danielahdez for reporting back and closing the issue :+1:

**Comment by YuriiMotov** [comment]
https://github.com/fastapi/fastapi/issues/973
